src/app/page.tsx

```diff
@@ -0,0 +1,9 @@
+import { NextPage } from 'next';
+
+export const runtime = 'experimental-edge';
```

Comment:
Can you pls also add a comment explaining the reasoning behind using this runtime, for future devs?

Reply:
I removed it